packages/schematics/src/collection/cypress-project/files/cypress.json

```diff
@@ -3,7 +3,7 @@
   "fixturesFolder": "<%= offsetFromRoot %>dist/out-tsc/<%= projectRoot %>/src/fixtures",
   "integrationFolder": "<%= offsetFromRoot %>dist/out-tsc/<%= projectRoot %>/src/integration",
   "pluginsFile": "<%= offsetFromRoot %>dist/out-tsc/<%= projectRoot %>/src/plugins/index.js",
-  "supportFile":  "<%= offsetFromRoot %>dist/out-tsc/<%= projectRoot %>/src/support/index.js",
+  "supportFile":  false,
```

Comment:
This relates to multiple tickets from the same Cypress bug:

- https://github.com/cypress-io/cypress/issues/2595
- https://github.com/cypress-io/cypress/issues/2270
- https://github.com/cypress-io/cypress/issues/1704

We put `false` until it will get a fix.